htmd/parameterization/ffmolecule.py

```diff
@@ -126,33 +126,30 @@ def _rename_mol(self):
         # This fixes up the atom naming and reside name to be consistent
         # NB this scheme matches what MATCH does. Don't change it
         # Or the naming will be inconsistent with the RTF
-        import re
 
-        hh = dict()
+        sufices = dict()
 
+        print('\nRename atoms:')
```

Comment:
Any specific reason why you use print instead of a logger as we do in the rest of HTMD?

Reply:
So far, loggers nowhere to be seen in `parameterize`...

Reply:
This is just a bug fix not a rewriting.

Reply:
Ok, it was just a general consideration. Maybe we should make an issue on it to move prints to loggers once the main refactoring is done